paml/paml.ttl

```diff
@@ -176,13 +176,21 @@ paml:ActivityNodeExecution rdf:type owl:Class ;
         rdfs:subClassOf sbol:Identified ,
     [ rdf:type owl:Restriction ; owl:allValuesFrom uml:ActivityNode ; owl:onProperty paml:node ;
       owl:minCardinality "1"^^xsd:nonNegativeInteger ; owl:maxCardinality "1"^^xsd:nonNegativeInteger ; ] ,
+    [ rdf:type owl:Restriction ; owl:allValuesFrom uml:LiteralInteger ; owl:onProperty paml:executionIndex ;
```

Comment:
What is the purpose of this field?
Note that executions may not be linearizable into a single order, given the availability of hardware to carry out actions in parallel.

Reply:
Executions are ordered because it assumes a non-deterministic actor that will pick the next enabled node.  This just records what order was chosen.

Reply:
This is true only for executions that have precisely one actor (which happen to be the cases we've worked with so far). Executions that use multiple actors will produce partial orders rather than total orders. So if you want to record an order, I would suggest using a partial order for doing so.

Two examples of partial order implementations:
- SBOL Constraint objects with ordering constraints
- PROV-O wasDerived from links

Reply:
Hmm, I really just needed this for the plotting, but I can live without it.  When we need a more general solution, I think your suggestions are perfect.